test/steps/openshift.go

```diff
@@ -42,14 +46,20 @@ func registerOpenShiftSteps(ctx *godog.ScenarioContext, data *Data) {
 // Build steps
 func (data *Data) startBuildFromExampleServicePath(buildName, localExamplePath string) error {
 	examplesRepositoryPath := data.KogitoExamplesLocation
-	_, err := framework.CreateCommand("oc", "start-build", buildName, "--from-dir="+examplesRepositoryPath+"/"+localExamplePath, "-n", data.Namespace).WithLoggerContext(data.Namespace).Execute()
-	return err
+	return framework.WaitForOnOpenshift(data.Namespace, fmt.Sprintf("Build '%s' to start", buildName), defaultTimeoutToStartBuildInMin,
+		func() (bool, error) {
+			_, err := framework.CreateCommand("oc", "start-build", buildName, "--from-dir="+examplesRepositoryPath+"/"+localExamplePath, "-n", data.Namespace).WithLoggerContext(data.Namespace).Execute()
```

Comment:
Hey @Sgitario!

Quick question: while do we need to manually start the build like this?

Reply:
It's needed until https://issues.redhat.com/browse/KOGITO-2161 is done. In this scenario, we're deploying from binaries (more in [this comment](https://issues.redhat.com/browse/KOGITO-2161?focusedCommentId=14159830&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14159830)).

Reply:
@Sgitario Can you just add the issue in comment ?

Reply:
The issue was already added by Karel in the scenario as a comment.